### PR TITLE
[will/popup] add padding and fit-content

### PR DESCRIPTION
**Pull request by WillHHH**

### CODE REVIEW

File: src/components/Popup/index.stories.tsx [hunk 1]
```diff
@@ -140,7 +140,9 @@ export const PopupContent = () => {
       <Popup
         isActive={centerPopupB}
         setActive={setCenterPopupB}
+        size={['', '700px']}
```
Comment (mxdi9i7) [inline src/components/Popup/index.stories.tsx:143]: no array, use object

File: src/components/Popup/index.tsx [hunk 2]
```diff
@@ -56,8 +57,16 @@ const Popup = ({
     Object.assign(popupProps, {
       style: {
         ...popupProps.style,
-        width: size[0],
-        height: size[1]
+        width: size[0] && size[0],
```
Comment (mxdi9i7) [inline src/components/Popup/index.tsx:60]: no array, use object

File: src/components/Popup/index.tsx [hunk 3]
```diff
@@ -56,8 +57,16 @@ const Popup = ({
...
+        height: size[1] && size[1]
```
Comment (mxdi9i7) [inline src/components/Popup/index.tsx:61]: no array, use object

File: src/components/Popup/index.stories.tsx (hunk 1 above)
Comment (WillHHH) [reply src/components/Popup/index.stories.tsx]: fixed

File: src/components/Popup/index.tsx (hunk 3 above)
Comment (WillHHH) [reply src/components/Popup/index.tsx]: fixed

File: src/components/Popup/index.tsx (hunk 2 above)
Comment (WillHHH) [reply src/components/Popup/index.tsx]: fixed

**Pull request merged**